test/data/sandbox/custom_worker_helper.js

```diff
@@ -0,0 +1,11 @@
+const Helper = require('../../../lib/helper');
+const assert = require('assert');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`assert` import should occur before import of `../../../lib/helper`](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4759786)